shared/codec/codec.go

```diff
@@ -48,7 +48,9 @@ func (p *ProtoCodec) FromAny(any *anypb.Any) (proto.Message, error) {
 	return msg, nil
 }
 
-// TODO(andrew): Retrieve this from the utility module via the application specific bus
+// DISCUSS: Retrieve this from the utility module via the application specific bus?
+// There are some parts of the code that does not have access to the bus;
```

Comment:
Ideally everything will have access to the bus via some sort of DI approach. We'll get there\